src/tfr_utilities/include/tfr_utilities/placeholder_utility.h

```diff
@@ -1,9 +1,10 @@
+
 #ifndef PLACEHOLDER_UTILITY_H
 #define PLACEHOLDER_UTILITY_H
 
 namespace tfr_utilities
 {
-    int Foo();
+        int Foo();
 }
 
-#endif
\ No newline at end of file
+#endif
```

Comment:
Is this file necessary for now? I'd personally love to save this until we know we need it (although that point is up for debate).

Reply:
I would say no, probably; we can delete this if we want.

Reply:
Oh this is actually a pretty huge change it causes all the remaining placeholder code not compile anymore, let's make that a seperate pull request